src/components/LanguageButtons.tsx

```diff
@@ -39,12 +39,12 @@ const ActiveLang = css`
 const LanguageButtons = () => {
   const fontSizes = [1, 2, 3, 4];
 
-  const [activeLang, setActiveLang] = useState(i18next.language);
+  const [activeLang, setActiveLang] = useState();
 
   const onLangClick = useCallback(
     (countryId: string) => () => {
       i18next.changeLanguage(countryId);
-      setActiveLang(countryId);
+      setActiveLang(activeLang);
```

Comment:
this shouldnt be changed, otherwise when you click on a language button the language will be set to the current active lang instead of the new one

Reply:
I didn't understand when I first did this but now I do.

The typescript error is fixed.